src/libdrakvuf/libdrakvuf.h

```diff
@@ -201,6 +201,7 @@ typedef struct drakvuf_trap_info
     {
         const cpuid_event_t* cpuid; /* For CPUID traps */
         const debug_event_t* debug; /* For DEBUG traps */
+        const reg_event_t*   reg;   /* For MSR traps */
```

Comment:
I think calling this field `msr` here would be more intuitive.

Reply:
Right now a user would access msr in a callback like this:
```c++
auto msr = info->reg->msr;
```
IMO this variant looks better, but I'm ok with the change.

Reply:
Yea, I guess it's fine as-is.